__tests__/app.test.js

```diff
@@ -118,39 +118,105 @@ describe('GET requests', () => {
             })
         })
     })
+    describe('/api/articles/:article_id/comments', () => {
+        describe('GET - status 200 - responds with an array of comment objects', () => {
+            test('responds with an array of comments for the given article_id of which each comment should have "comment_id", "votes", "created_at", "author", "body" and "article_id" properties', () => {
+                return request(app)
+                    .get('/api/articles/1/comments')
+                    .expect(200)
+                    .then(({ body: { comments } }) => {
+                        comments.forEach(comment => {
```

Comment:
Please check the length of the array first! Skipping that step can lead to a false positive test.